### ISSUE TITLE
AE2 crafting errors

### ISSUE THREAD

**Issue by Rainmaker709** [FTBTeam/FTB-Modpack-Issues#5156]
### Modpack

FTB Skies Expert

### Modpack version

1.7.0

### Describe the bug

With the Quantum Assembler, many times when you request a bulk craft, it will send between 30% to 60% of the ingredients and then just never send more so craft is stuck "crafting".

With the Metallurgic Infuser, all recipes only send half of stated ingredients. Same issue.

Example: Request 10 Conductive Soldering Alloy. This would be 5 sets of ingredients. System states correct amount on confirmation screen. When you click Start, system sends 3 sets of ingredients. After those 3 finish, system sits idle waiting for the remaining ingredients. Craft stays in queue until cancelled.

Does not affect other Mekanism machines such as Enrichment Chamber or other Custom Machines such as Fabrication Matrix.

Note that the Quantum Assembler is being accessed via a Entangled Block as having a machine that requires 3 inputs but only leaving ONE block face accessible is stupid AF.

### Steps to reproduce

See description. 

Mid game bug requiring full AE2 system, basic Mekanism, progression required Custom Machines.

### Expected behaviour

When requesting craft of X, to receive X.

**Comment by isochronous** [comment]
Do you have blocking mode turned on?

**Comment by Rainmaker709** [comment]
The bug was only when the Provider is allowed unrestricted input. I put down the pack at the beginning of the week and after firing it up today, I can't replicate this bug.

I had bypassed it by using "Lock until result returned" and I had tried relogging at the time with no success but it seems to no longer happen. I'll try unlocking all my providers and reopen if it happens again but until then y'all can mark it as resolved for now if you want. :/
